### ISSUE TITLE
Error not handle on THEIA node server

### ISSUE THREAD

**Issue by lmcbout** [theia-ide/theia#867]
I transferred an existing git repo I have into THEIA and I got many error like the following one, error not handle: 

The command `git status --untracked-files=all --branch --porcelain=2 -z` exited with an unexpected code: 128. The caller should either handle this error, or expect that exit code.
fatal: Not a git repository (or any of the parent directories): .git
(The error was parsed as 27: This is not a git repository.)

**Comment by marcdumais-work** [comment]
When you say "transferred", do you mean that you dragged the git repo in Theia's window? If so the issue might be that the git extension does not expect that a partial (at least at first, during the transfer) git repo can appear, and starts right away to issue "git status" commands on it, unsuccessfully. Not sure how much we can do about that...

**Comment by lmcbout** [comment]
I meant "DRAG" as you said, but when dragging a smaller git repo, I didn't get the error, I think we should be able to restrict the error so the user web application continue working, not freezing

**Comment by kittaakos** [comment]
We should log these problems a more consumable way. See: #759?

**Comment by marcdumais-work** [comment]
I was able to reproduce this, outside Theia, just using plain git. However I do not think it's valid, or at least that there is much we can do about it. 

Git seems smart enough that if one runs "git status" on a repo that is being cloned, it's handled gracefully (exit code 0). However if we do "git status" on a repo that is being copied file-per-file (e.g. cp -a repo /someotherplace/), then git might find that the files copied so far do not seem to constitute a valid git repo, at that moment:

```
fatal: Not a git repository (or any parent up to mount point /project)
```

So, IMHO, as long as we fall back on our feet, once the repo is finished copying correctly, I think it's allright. If the repo is somehow not copied correctly, then that's another issue.
